packages/wonder-blocks-dropdown/components/multi-select.js

```diff
@@ -352,12 +396,22 @@ export default class MultiSelect extends React.Component<Props, State> {
             </SelectOpener>
         );
 
+        const shortcutItems = this.getShortcuts(numOptions);
+        const filteredMenuItems = this.getMenuItems(allChildren);
+
         return (
             <DropdownCore
                 role="listbox"
                 alignment={alignment}
                 dropdownStyle={[selectDropdownStyle, dropdownStyle]}
-                items={items}
+                handleSearchTextChanged={
+                    isFilterableByLabel ? this.handleSearchTextChanged : null
+                }
+                items={
+                    isFilterableByLabel
+                        ? filteredMenuItems
+                        : [...shortcutItems, ...filteredMenuItems]
```

Comment:
We could inline the call to `this.getShortcuts(numOptions)` here to avoid calling it when we don't need to.